app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({name: self})
+
+
+def create_person_list(peoples: list) -> list:
+    main_list = []
+
+    for people in peoples:
+        person = Person(people["name"], people["age"])
+        main_list.append(person)
```

Comment:
You could directly append a class instance during its initialization here, there is no need for a person variable here.